runtime/mangata-rococo/src/lib.rs

```diff
@@ -1559,6 +1559,36 @@ impl pallet_proxy::Config for Runtime {
 	type AnnouncementDepositFactor = AnnouncementDepositFactor;
 }
 
+parameter_types! {
+	// Add one item in storage and take 258 bytes
+	pub const BasicDeposit: Balance = deposit(1, 256);
+	// No item in storage but takes 66 bytes
+	pub const FieldDeposit: Balance = deposit(0, 66);
+	// No item in storage but takes 53 bytes
+	pub const SubAccountDeposit: Balance = deposit(0, 53);
+	pub const MaxSubAccounts: u32 = 100;
+	pub const MaxAdditionalFields: u32 = 100;
```

Comment:
if you could provide some docs what these params are about that would be very helpful for the future

Reply:
all of these are pretty nicely documented in the identity pallet, i think that should suffic
i can add some comments for the specific values thou